packages/distributor/solana/client.ts

```diff
@@ -180,7 +180,9 @@ export default class SolanaDistributorClient {
       ixs.push(newClaim(args, accounts, this.programId));
     }
 
-    ixs.push(claimLocked(accounts, this.programId));
+    if (data.amountLocked.gt(new BN("0"))) {
```

Comment:
just wanna put it here, as an option

less transformations
```suggestion
    if (data.amountLocked.gtn(0)) {
```